tests/unit/cirrus/publisher_plugins_tests.py

```diff
@@ -0,0 +1,137 @@
+import __builtin__
+import json
+import mock
+import os
+import tempfile
+import unittest
+
+import pluggage.registry
+
+from cirrus.documentation_utils import doc_artifact_name
+
+from harnesses import CirrusConfigurationHarness, write_cirrus_conf
+
+
+class FileServerPublisherTests(unittest.TestCase):
```

Comment:
document test classes and test methods with brief desc of what you are covering and description of what you are testing